server/app/__init__.py

```diff
@@ -4,7 +4,9 @@
     from ConfigParser import ConfigParser, NoSectionError, NoOptionError
 
 import os
+
 from flask import Flask
+from flask_caching import Cache
```

Comment:
Let's talk about it. The problem with caching is that the files are not stored anywhere, so after whatever the caching time is over (right now 5 mins) the charts will have to be regenerated. I would very much prefer to write the generated files to disk. This means that over time there would be less and less data to generate. Also, if the site ever becomes popular, this will scale better. Disk space is cheap these days.

Reply:
After discussion with you I change the solution. Now the plots are created during receiving. I prepared script which create plots for old data.

Reply:
Thanks!